### ISSUE TITLE
Support for different domains

### ISSUE THREAD

**Issue by OndrejVasicek** [codemix/yii2-localeurls#157]
Hi mikehaertl and thanks for your great extensions. I’ve already used your extensions on several projects, but on the current project, we have two languages and for every language there is a dedicated domain. Like example.com and example.cz. How should I set the localeurls with this in mind? Or is it a matter of url rules? I thought it has to be in management of this extension, because the extension handles the “language” param and creates different version of url.

Thanks for your answer.

**Comment by mikehaertl** [comment]
This exention is really only for languages that are part of the URL, not for host names. In your case you could probably simply check `$_SERVER['HTTP_HOST']` in your config file and set the app language from a simple map (hostname => language, ...).

**Comment by OndrejVasicek** [comment]
Thank you mikehaertl, I understand. I chose a different approach. Rewriting language by force doesn’t suit your extension. It creates a wrong url, depending the place (time) I forced the language param. So I just decided to keep the plugin handle the language param and creating urls and I just redirect the url in some rare cases, when language param doesn’t correspond with the HTTP_HOST. Works fine.
Thank you for your advice.
